src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Raven.Imports.Newtonsoft.Json;
+using Raven.Server.Commercial;
+using Raven.Server.Routing;
+
+namespace Raven.Server.Web.Studio
+{
+    public class LicenseHandler : RequestHandler
+    {
+        [RavenAction("/license/status", "GET")]
+        public Task Status()
+        {
+            HttpContext.Response.ContentType = "application/json";
+            HttpContext.Response.StatusCode = 200;
+            return HttpContext.Response.WriteAsync(JsonConvert.SerializeObject(Commercial.LicenseHandler.GetLicenseStatus()));
```

Comment:
Don't write code like that, use the standard means for generating JSON on 4.0, blittable.
In this case, using `DynamicJsonObject` to generate the object to write.